### PR TITLE
[Term Entry] Swift dictionary.mapValues()

### LINKED ISSUE #5123: [Term Entry] Swift dictionaries mapValues()
### Reason/inspiration (optional)

"We would like a new term entry on the `mapValues` for Dictionaries in Swift. The entry should go in a new file under `content/swift/concepts/dictionaries/terms/mapValues/mapValues.md`
The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```swift ``` codeblock and ```shell ``` that shows the example 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Great-Visions-Code**
### Description

- Add .mapValues() term entry for Swift

### Issue Solved

- Closes #5123 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/swift/concepts/dictionaries/terms/mapValues/mapValues.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- The `mapValues` method returns a new dictionary with the same keys, but with the values transformed based on the provided closure.
+- `transformation`: A closure that defines how each value in the dictionary should be transformed.
+
+The result of `mapValues()` is a new dictionary with the same keys, but the values are transformed based on the provided closure.
+
+> **Note:** `mapValues()` does not modify the original dictionary; instead, it returns a new dictionary with the transformed values.
+
+## Example
+
+In the following example, the `mapValues()` method is used to increase each student's grade by 10 points in the `studentGrades` dictionary:
```
Comment (cigar-galaxy82) [inline content/swift/concepts/dictionaries/terms/mapValues/mapValues.md:34]:
```suggestion
In the following example, the `mapValues()` method is used to increase each student's grade by `10` points while keeping the keys unchanged in the `studentGrades` dictionary:
```

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @Great-Visions-Code thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

**Comment by cigar-galaxy82** [comment]
Hi, @Great-Visions-Code thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

**Comment by Great-Visions-Code** [comment]
@cigar-galaxy82 I have made the suggested changes. Please let me know if anything else is needed. Thank you,

**Review by cigar-galaxy82 — APPROVED** [review]

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @Great-Visions-Code 😄 

The entry is ready to be merged! 🚀

**Pull request merged**